### DISCUSSION TITLE
Computing the value of Discharge Capacity [Ah]?

### DISCUSSION THREAD

**Discussion by optibikash** [pybamm-team/PyBaMM#1306] [Q&A] [answered]
The output of the cyclic aging of the battery done using cycling_ageing_yang.py plots Discharge Capacity [Ah]. How is the value computed in the package? Can I be directed to the file which computes the value of Discharge Capacity [Ah]?

**Comment by brosaplanella** [comment, marked as answer]
It is calculated in `pybamm/models/submodels/external_circuit/base_external_circuit.py` where the variable `"Discharge capacity [A.h]"` is defined. The computation is done via Coulomb counting defined in

```python3
def set_rhs(self, variables):
    # ODE for discharge capacity
    Q = variables["Discharge capacity [A.h]"]
    I = variables["Current [A]"]
    self.rhs[Q] = I * self.param.timescale / 3600
```

**Reply by optibikash** [reply]
Thanks, Ferran for the answer. I also implemented Coulomb counting and got the same results. Just thought of comparing and happy to see the same being already used.
